app/src/main/java/com/example/demomvvm/screen/listUnsplash/adapter/ItemUnsplashViewModel.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.demomvvm.screen.listUnsplash.adapter
+
+import androidx.databinding.BaseObservable
+import androidx.databinding.Bindable
+import com.example.demomvvm.data.model.Unsplash
+
+class ItemUnsplashViewModel() : BaseObservable() {
+
+    @Bindable
+    var unsplash: Unsplash? = null
+        set(value) {
+            field = value
+            notifyChange()
```

Comment:
```suggestion
            notifyChange()
```
Này nó dùng làm gì nhỉ